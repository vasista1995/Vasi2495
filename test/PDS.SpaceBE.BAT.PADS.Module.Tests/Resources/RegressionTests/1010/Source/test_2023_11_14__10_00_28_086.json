{
  "IdSource": "SPACEACT2:BAT:BE:RU342054B04:BE1I:FOL_Autovision:A2PLWirePullTop:20375:239203540:1",
  "Data_flat_Metadata": {
    "Device": "S0946T",
    "ParameterClass": "PPAP",
    "CFComment": "FOL",
    "Segment": "A2PL",
    "F56Parameter": "N",
    "FourDReport": "*",
    "ParameterName": "A2PLWirePullTop",
    "ParameterUnit": "g",
    "ParameterOper": "FOL_Autovision",
    "ParameterFacility": "BE1I",
    "SiteKey": "BAT",
    "ChannelId": "20375",
    "SpaceInstanceName": "BE",
    "CreatedTimestampUtc": "2023-11-03T11:26:34Z",
    "CreatedTimestamp": "2023-11-03T18:26:34Z",
    "UpdatedTimestamp": "2023-11-14T15:00:09Z",
    "UpdatedTimestampUtc": "2023-11-14T08:00:09Z",
    "ExportedTimestampUtc": "2023-11-03T18:26:34Z",
    "ExportedTimestamp": "2023-11-03T18:26:34Z",
    "SampleTimestampUtc": "2023-11-03T11:26:34Z",
    "SampleTimestamp": "2023-11-03T18:26:34Z",
    "SourceDataLevel": "C",
    "ChannelName": "Wire_PullForce/CU/50um/DS20/A2CU",
    "ChannelDescr": "A2PLWirePullTop\\DS20\\A2CU\\50 ",
    "RvStoreFlag": "Y",
    "CalcCreated": "09/22/2022 00:46:42",
    "ILdsID": 0,
    "AcceptFlag": "Y",
    "UpdatedBy": "iscw8cc2082mwr",
    "ClOrigin": 0,
    "OutOfOrder": "N",
    "Calc2ID": 1893317,
    "ViolCount": 1,
    "CalculationStrategy": "ND",
    "LdsID": "1010",
    "IsCalcParam": 0,
    "ChannelType": "N",
    "ChState": "Online"
  },
  "Data_1list_RawValues": {
    "Data_flat_Limits": {
      "CkcId": "1",
      "MeasurementSpecLimits": {
        "SpecHighEnabled": "N",
        "SpecLowEnabled": "Y",
        "SpecLow": 10.0,
        "SpecTarget": 20.0,
        "SpecTargetOrigin": "V",
        "ExtSpecLimEnable": "NN",
        "ExtSpecUpper": 0.0,
        "ExtSpecLower": 0.0,
        "LimitEnable": "YYNYNNYNYYYYYYYYYYYNNNNNNNNNNNNNNNNNN"
      },
      "SpaceAggregates": {
        "Mean": "42,61",
        "Median": "42,358",
        "Sigma": ",645670636957089",
        "Min": "41,938",
        "Max": "43,575",
        "Q1": "42,1375",
        "Q3": "43,01175"
      },
      "ControlLimits": {
        "ctrl_mean_low": 33.3151540719785,
        "CtrlMeanLowEnabled": "Y",
        "ctrl_mean_target": 44.0952438837921,
        "CtrlMeanTargetEnabled": "Y",
        "ctrl_mean_high": 54.8753336956056,
        "CtrlMeanHighEnabled": "Y",
        "CtrlLowEnabled": "N",
        "CtrlSigmaCenterEnabled": "N",
        "CtrlRangeCenterEnabled": "Y",
        "CtrlCenterEnabled": "N",
        "CtrlHighEnabled": "N",
        "CtrlSigmaLowEnabled": "N",
        "CtrlSigmaHighEnabled": "N",
        "CtrlRangeLowEnabled": "N",
        "ctrl_range_target": 5.51093157534837,
        "ctrl_range_high": 23.5050462728126,
        "CtrlRangeHighEnabled": "Y",
        "ExtMaMV": 42.214515625,
        "ExtEwmaMV": 42.1600609177337,
        "ExtMsMV": 0.58919672542369,
        "EwmaS": 1.05628196822843,
        "EwmaR": 3.03607050173468,
        "ExtEwmaMVLCL": 40.49776600118,
        "ExtEwmaMVCenter": 44.0952438837921,
        "ExtEwmaMVUCL": 47.6927217664041,
        "ExtMaMVLCL": 41.5514428778285,
        "ExtMaMVCenter": 44.0952438837921,
        "ExtMaMVUCL": 46.6390448897557,
        "EwmaSLCL": 0.867805417770263,
        "EwmaSCenter": 2.16204504781218,
        "EwmaSUCL": 3.4562846778541,
        "EwmaRLCL": 2.32059459732032,
        "EwmaRCenter": 5.71875688073395,
        "EwmaRUCL": 9.11691916414757,
        "ExtMSMVLCL": 0.00392234796868616,
        "ExtMSMVCenter": 1.84966154041296,
        "ExtMSMVUCL": 7.43016203913499,
        "MVLal": 0.0,
        "MVUal": 0.0,
        "RawLal": 29.7000316153901,
        "RawUal": 58.490456152194,
        "SigmaLal": 0.0,
        "SigmaUal": 0.0,
        "RangeLal": 0.0,
        "RangeUal": 0.0
      }
    },
    "Data_1list_RawValues": [
      {
        "SampleId": 239203540,
        "Seqnr": 1,
        "Value": 41.938,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 1,
        "ViolationComments": "RC   : A2PL WIRE PULL TOP 8 POINT RUN DOWN\t\nCA   : UNDER MONITORING BY QE/UPE",
        "ViolationList": "10",
        "PrimaryViolation": "10",
        "PrimaryViolationComments": "RC   : A2PL WIRE PULL TOP 8 POINT RUN DOWN\t\nCA   : UNDER MONITORING BY QE/UPE"
      },
      {
        "SampleId": 239203540,
        "Seqnr": 2,
        "Value": 42.257,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 1,
        "ViolationComments": "RC   : A2PL WIRE PULL TOP 8 POINT RUN DOWN\t\nCA   : UNDER MONITORING BY QE/UPE",
        "ViolationList": "10",
        "PrimaryViolation": "10",
        "PrimaryViolationComments": "RC   : A2PL WIRE PULL TOP 8 POINT RUN DOWN\t\nCA   : UNDER MONITORING BY QE/UPE"
      },
      {
        "SampleId": 239203540,
        "Seqnr": 3,
        "Value": 43.575,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 1,
        "ViolationComments": "RC   : A2PL WIRE PULL TOP 8 POINT RUN DOWN\t\nCA   : UNDER MONITORING BY QE/UPE",
        "ViolationList": "10",
        "PrimaryViolation": "10",
        "PrimaryViolationComments": "RC   : A2PL WIRE PULL TOP 8 POINT RUN DOWN\t\nCA   : UNDER MONITORING BY QE/UPE"
      },
      {
        "SampleId": 239203540,
        "Seqnr": 8,
        "Value": 43.551,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 1,
        "ViolationComments": "RC   : A2PL WIRE PULL TOP 8 POINT RUN DOWN\t\nCA   : UNDER MONITORING BY QE/UPE",
        "ViolationList": "10",
        "PrimaryViolation": "10",
        "PrimaryViolationComments": "RC   : A2PL WIRE PULL TOP 8 POINT RUN DOWN\t\nCA   : UNDER MONITORING BY QE/UPE"
      },
      {
        "SampleId": 239203540,
        "Seqnr": 5,
        "Value": 42.141,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 1,
        "ViolationComments": "RC   : A2PL WIRE PULL TOP 8 POINT RUN DOWN\t\nCA   : UNDER MONITORING BY QE/UPE",
        "ViolationList": "10",
        "PrimaryViolation": "10",
        "PrimaryViolationComments": "RC   : A2PL WIRE PULL TOP 8 POINT RUN DOWN\t\nCA   : UNDER MONITORING BY QE/UPE"
      },
      {
        "SampleId": 239203540,
        "Seqnr": 6,
        "Value": 42.459,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 1,
        "ViolationComments": "RC   : A2PL WIRE PULL TOP 8 POINT RUN DOWN\t\nCA   : UNDER MONITORING BY QE/UPE",
        "ViolationList": "10",
        "PrimaryViolation": "10",
        "PrimaryViolationComments": "RC   : A2PL WIRE PULL TOP 8 POINT RUN DOWN\t\nCA   : UNDER MONITORING BY QE/UPE"
      },
      {
        "SampleId": 239203540,
        "Seqnr": 7,
        "Value": 42.127,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 1,
        "ViolationComments": "RC   : A2PL WIRE PULL TOP 8 POINT RUN DOWN\t\nCA   : UNDER MONITORING BY QE/UPE",
        "ViolationList": "10",
        "PrimaryViolation": "10",
        "PrimaryViolationComments": "RC   : A2PL WIRE PULL TOP 8 POINT RUN DOWN\t\nCA   : UNDER MONITORING BY QE/UPE"
      },
      {
        "SampleId": 239203540,
        "Seqnr": 4,
        "Value": 42.832,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 1,
        "ViolationComments": "RC   : A2PL WIRE PULL TOP 8 POINT RUN DOWN\t\nCA   : UNDER MONITORING BY QE/UPE",
        "ViolationList": "10",
        "PrimaryViolation": "10",
        "PrimaryViolationComments": "RC   : A2PL WIRE PULL TOP 8 POINT RUN DOWN\t\nCA   : UNDER MONITORING BY QE/UPE"
      }
    ],
    "SampleSize": 8
  },
  "ProductionAction": {
    "Id": "SPACEACT1:BAT:BE:BE1I:FOL_Autovision:A2PLWirePullTop:20375",
    "Type": "SpaceAction"
  },
  "Item": {
    "Id": "Measlot:",
    "IdSystemName": "Measlot",
    "Type": "Lot"
  },
  "SystemLog": {
    "ExtractedTimestampUtc": "0001-01-01T00:00:00",
    "EnqueuedTimestampUtc": "0001-01-01T00:00:00",
    "UniqueSourceKey": "BAT:BE:RU342054B04:BE1I:FOL_Autovision:A2PLWirePullTop:20375:239203540:1"
  }
}