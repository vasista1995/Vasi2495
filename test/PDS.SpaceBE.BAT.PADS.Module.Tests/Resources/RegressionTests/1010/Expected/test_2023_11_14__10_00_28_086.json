{
  "Data_flat_Metadata": {
    "ParameterOper": "FOL_Autovision",
    "ParameterFacility": "BE1I",
    "SiteKey": "BAT",
    "SpaceInstanceName": "BE",
    "BeginTimestampUtc": "2023-11-03T11:26:34Z",
    "BeginTimestamp": "2023-11-03T18:26:34Z",
    "EndTimestampUtc": "2023-11-03T11:26:34Z",
    "EndTimestamp": "2023-11-03T18:26:34Z"
  },
  "Data_flat_StrangeMetadata": {
    "Device": "S0946T",
    "ParameterClass": "PPAP",
    "Segment": "A2PL",
    "F56Parameter": "N"
  },
  "Data_1list_Parameters": [
    {
      "Data_1list_RawValues": [
        {
          "Seqnr": 1,
          "SampleId": 239203540,
          "Value": 41.938,
          "is_flagged": "N",
          "SampleTimestamp": "2023-11-03T18:26:34Z",
          "SampleTimestampUtc": "2023-11-03T11:26:34Z",
          "CreatedTimestamp": "2023-11-03T18:26:34Z",
          "CreatedTimestampUtc": "2023-11-03T11:26:34Z",
          "UpdatedTimestamp": "2023-11-14T15:00:09Z",
          "UpdatedTimestampUtc": "2023-11-14T08:00:09Z",
          "NumViolations": 1,
          "ViolationComments": "RC   : A2PL WIRE PULL TOP 8 POINT RUN DOWN  CA   : UNDER MONITORING BY QE UPE",
          "ViolationList": "10",
          "PrimaryViolation": "10",
          "PrimaryViolationComments": "RC   : A2PL WIRE PULL TOP 8 POINT RUN DOWN  CA   : UNDER MONITORING BY QE UPE",
          "SampleMean": 42.61,
          "SampleMin": 41.938,
          "SampleMax": 43.575,
          "SampleMedian": 42.358,
          "SampleStdev": 0.645670636957089,
          "SampleQ1": 42.1375,
          "SampleQ3": 43.01175,
          "SampleSize": 8
        },
        {
          "Seqnr": 2,
          "SampleId": 239203540,
          "Value": 42.257,
          "is_flagged": "N",
          "SampleTimestamp": "2023-11-03T18:26:34Z",
          "SampleTimestampUtc": "2023-11-03T11:26:34Z",
          "CreatedTimestamp": "2023-11-03T18:26:34Z",
          "CreatedTimestampUtc": "2023-11-03T11:26:34Z",
          "UpdatedTimestamp": "2023-11-14T15:00:09Z",
          "UpdatedTimestampUtc": "2023-11-14T08:00:09Z",
          "NumViolations": 1,
          "ViolationComments": "RC   : A2PL WIRE PULL TOP 8 POINT RUN DOWN  CA   : UNDER MONITORING BY QE UPE",
          "ViolationList": "10",
          "PrimaryViolation": "10",
          "PrimaryViolationComments": "RC   : A2PL WIRE PULL TOP 8 POINT RUN DOWN  CA   : UNDER MONITORING BY QE UPE",
          "SampleMean": 42.61,
          "SampleMin": 41.938,
          "SampleMax": 43.575,
          "SampleMedian": 42.358,
          "SampleStdev": 0.645670636957089,
          "SampleQ1": 42.1375,
          "SampleQ3": 43.01175,
          "SampleSize": 8
        },
        {
          "Seqnr": 3,
          "SampleId": 239203540,
          "Value": 43.575,
          "is_flagged": "N",
          "SampleTimestamp": "2023-11-03T18:26:34Z",
          "SampleTimestampUtc": "2023-11-03T11:26:34Z",
          "CreatedTimestamp": "2023-11-03T18:26:34Z",
          "CreatedTimestampUtc": "2023-11-03T11:26:34Z",
          "UpdatedTimestamp": "2023-11-14T15:00:09Z",
          "UpdatedTimestampUtc": "2023-11-14T08:00:09Z",
          "NumViolations": 1,
          "ViolationComments": "RC   : A2PL WIRE PULL TOP 8 POINT RUN DOWN  CA   : UNDER MONITORING BY QE UPE",
          "ViolationList": "10",
          "PrimaryViolation": "10",
          "PrimaryViolationComments": "RC   : A2PL WIRE PULL TOP 8 POINT RUN DOWN  CA   : UNDER MONITORING BY QE UPE",
          "SampleMean": 42.61,
          "SampleMin": 41.938,
          "SampleMax": 43.575,
          "SampleMedian": 42.358,
          "SampleStdev": 0.645670636957089,
          "SampleQ1": 42.1375,
          "SampleQ3": 43.01175,
          "SampleSize": 8
        },
        {
          "Seqnr": 8,
          "SampleId": 239203540,
          "Value": 43.551,
          "is_flagged": "N",
          "SampleTimestamp": "2023-11-03T18:26:34Z",
          "SampleTimestampUtc": "2023-11-03T11:26:34Z",
          "CreatedTimestamp": "2023-11-03T18:26:34Z",
          "CreatedTimestampUtc": "2023-11-03T11:26:34Z",
          "UpdatedTimestamp": "2023-11-14T15:00:09Z",
          "UpdatedTimestampUtc": "2023-11-14T08:00:09Z",
          "NumViolations": 1,
          "ViolationComments": "RC   : A2PL WIRE PULL TOP 8 POINT RUN DOWN  CA   : UNDER MONITORING BY QE UPE",
          "ViolationList": "10",
          "PrimaryViolation": "10",
          "PrimaryViolationComments": "RC   : A2PL WIRE PULL TOP 8 POINT RUN DOWN  CA   : UNDER MONITORING BY QE UPE",
          "SampleMean": 42.61,
          "SampleMin": 41.938,
          "SampleMax": 43.575,
          "SampleMedian": 42.358,
          "SampleStdev": 0.645670636957089,
          "SampleQ1": 42.1375,
          "SampleQ3": 43.01175,
          "SampleSize": 8
        },
        {
          "Seqnr": 5,
          "SampleId": 239203540,
          "Value": 42.141,
          "is_flagged": "N",
          "SampleTimestamp": "2023-11-03T18:26:34Z",
          "SampleTimestampUtc": "2023-11-03T11:26:34Z",
          "CreatedTimestamp": "2023-11-03T18:26:34Z",
          "CreatedTimestampUtc": "2023-11-03T11:26:34Z",
          "UpdatedTimestamp": "2023-11-14T15:00:09Z",
          "UpdatedTimestampUtc": "2023-11-14T08:00:09Z",
          "NumViolations": 1,
          "ViolationComments": "RC   : A2PL WIRE PULL TOP 8 POINT RUN DOWN  CA   : UNDER MONITORING BY QE UPE",
          "ViolationList": "10",
          "PrimaryViolation": "10",
          "PrimaryViolationComments": "RC   : A2PL WIRE PULL TOP 8 POINT RUN DOWN  CA   : UNDER MONITORING BY QE UPE",
          "SampleMean": 42.61,
          "SampleMin": 41.938,
          "SampleMax": 43.575,
          "SampleMedian": 42.358,
          "SampleStdev": 0.645670636957089,
          "SampleQ1": 42.1375,
          "SampleQ3": 43.01175,
          "SampleSize": 8
        },
        {
          "Seqnr": 6,
          "SampleId": 239203540,
          "Value": 42.459,
          "is_flagged": "N",
          "SampleTimestamp": "2023-11-03T18:26:34Z",
          "SampleTimestampUtc": "2023-11-03T11:26:34Z",
          "CreatedTimestamp": "2023-11-03T18:26:34Z",
          "CreatedTimestampUtc": "2023-11-03T11:26:34Z",
          "UpdatedTimestamp": "2023-11-14T15:00:09Z",
          "UpdatedTimestampUtc": "2023-11-14T08:00:09Z",
          "NumViolations": 1,
          "ViolationComments": "RC   : A2PL WIRE PULL TOP 8 POINT RUN DOWN  CA   : UNDER MONITORING BY QE UPE",
          "ViolationList": "10",
          "PrimaryViolation": "10",
          "PrimaryViolationComments": "RC   : A2PL WIRE PULL TOP 8 POINT RUN DOWN  CA   : UNDER MONITORING BY QE UPE",
          "SampleMean": 42.61,
          "SampleMin": 41.938,
          "SampleMax": 43.575,
          "SampleMedian": 42.358,
          "SampleStdev": 0.645670636957089,
          "SampleQ1": 42.1375,
          "SampleQ3": 43.01175,
          "SampleSize": 8
        },
        {
          "Seqnr": 7,
          "SampleId": 239203540,
          "Value": 42.127,
          "is_flagged": "N",
          "SampleTimestamp": "2023-11-03T18:26:34Z",
          "SampleTimestampUtc": "2023-11-03T11:26:34Z",
          "CreatedTimestamp": "2023-11-03T18:26:34Z",
          "CreatedTimestampUtc": "2023-11-03T11:26:34Z",
          "UpdatedTimestamp": "2023-11-14T15:00:09Z",
          "UpdatedTimestampUtc": "2023-11-14T08:00:09Z",
          "NumViolations": 1,
          "ViolationComments": "RC   : A2PL WIRE PULL TOP 8 POINT RUN DOWN  CA   : UNDER MONITORING BY QE UPE",
          "ViolationList": "10",
          "PrimaryViolation": "10",
          "PrimaryViolationComments": "RC   : A2PL WIRE PULL TOP 8 POINT RUN DOWN  CA   : UNDER MONITORING BY QE UPE",
          "SampleMean": 42.61,
          "SampleMin": 41.938,
          "SampleMax": 43.575,
          "SampleMedian": 42.358,
          "SampleStdev": 0.645670636957089,
          "SampleQ1": 42.1375,
          "SampleQ3": 43.01175,
          "SampleSize": 8
        },
        {
          "Seqnr": 4,
          "SampleId": 239203540,
          "Value": 42.832,
          "is_flagged": "N",
          "SampleTimestamp": "2023-11-03T18:26:34Z",
          "SampleTimestampUtc": "2023-11-03T11:26:34Z",
          "CreatedTimestamp": "2023-11-03T18:26:34Z",
          "CreatedTimestampUtc": "2023-11-03T11:26:34Z",
          "UpdatedTimestamp": "2023-11-14T15:00:09Z",
          "UpdatedTimestampUtc": "2023-11-14T08:00:09Z",
          "NumViolations": 1,
          "ViolationComments": "RC   : A2PL WIRE PULL TOP 8 POINT RUN DOWN  CA   : UNDER MONITORING BY QE UPE",
          "ViolationList": "10",
          "PrimaryViolation": "10",
          "PrimaryViolationComments": "RC   : A2PL WIRE PULL TOP 8 POINT RUN DOWN  CA   : UNDER MONITORING BY QE UPE",
          "SampleMean": 42.61,
          "SampleMin": 41.938,
          "SampleMax": 43.575,
          "SampleMedian": 42.358,
          "SampleStdev": 0.645670636957089,
          "SampleQ1": 42.1375,
          "SampleQ3": 43.01175,
          "SampleSize": 8
        }
      ],
      "CFComment": "FOL",
      "FourDReport": "*",
      "Data_flat_Limits": {
        "CKC_ID": "1",
        "MeasurementSpecLimits": {
          "SpecHighEnabled": "N",
          "SpecLowEnabled": "Y",
          "spec_low": 10.0,
          "target": 20.0
        },
        "ControlLimits": {
          "ctrl_mean_low": 33.3151540719785,
          "CtrlMeanLowEnabled": "Y",
          "ctrl_mean_target": 44.0952438837921,
          "CtrlMeanTargetEnabled": "Y",
          "ctrl_mean_high": 54.8753336956056,
          "CtrlMeanHighEnabled": "Y",
          "CtrlLowEnabled": "N",
          "CtrlHighEnabled": "N",
          "CtrlStdevLowEnabled": "N",
          "CtrlSigmaCenterEnabled": "N",
          "CtrlRangeCenterEnabled": "Y",
          "CtrlCenterEnabled": "N",
          "CtrlStdevHighEnabled": "N",
          "CtrlRangeLowEnabled": "N",
          "ctrl_range_target": 5.51093157534837,
          "ctrl_range_high": 23.5050462728126,
          "CtrlRangeHighEnabled": "Y"
        }
      },
      "Data_flat_MeasurementAggregates": {
        "exec_count": 8,
        "base_count": 8,
        "flagged_count": 0,
        "mean": 42.61,
        "median": 42.358000000000004,
        "stdev": 0.6039695770483814,
        "range": 1.6370000000000005,
        "min": 41.938,
        "max": 43.575,
        "q02": 41.96446,
        "q05": 42.00415,
        "q25": 42.1375,
        "q75": 43.011750000000006,
        "q95": 43.5666,
        "q98": 43.57164,
        "NumViolations": 1,
        "ViolationComments": "RC   : A2PL WIRE PULL TOP 8 POINT RUN DOWN  CA   : UNDER MONITORING BY QE UPE",
        "ViolationList": "10",
        "PrimaryViolation": "10",
        "PrimaryViolationComments": "RC   : A2PL WIRE PULL TOP 8 POINT RUN DOWN  CA   : UNDER MONITORING BY QE UPE",
        "Samples": "239203540"
      },
      "ParameterName": "A2PLWirePullTop",
      "ParameterUnit": "g",
      "ChannelId": "20375",
      "SourceDataLevel": "C",
      "channel_name": "Wire_PullForce CU 50um DS20 A2CU",
      "channel_description": "A2PLWirePullTop DS20 A2CU 50 ",
      "RvStoreFlag": "Y",
      "created_timestamp_utc": "2023-11-03T11:26:34Z",
      "created_timestamp": "2023-11-03T18:26:34Z",
      "updated_timestamp_utc": "2023-11-14T08:00:09Z",
      "updated_timestamp": "2023-11-14T15:00:09Z"
    }
  ],
  "Document": {
    "Version": "1.0",
    "Type": "ProcessControl",
    "Repetition": {
      "Id": "0",
      "IdBaseValue": "2023-11-14T08:00:09Z"
    }
  },
  "ProductionAction": {
    "Id": "SPACEAGGED2:BAT:BE:BE1I:FOL_Autovision:",
    "Type": "SpaceAggregation"
  },
  "Item": {
    "Id": "Measlot:",
    "IdSystemName": "Measlot",
    "Type": "Lot"
  },
  "SystemLog": {
    "UniqueSourceKey": "BAT:BE:RU342054B04:BE1I:FOL_Autovision:A2PLWirePullTop:20375:239203540:1",
    "ExtractedTimestampUtc": "0001-01-01T00:00:00",
    "EnqueuedTimestampUtc": "0001-01-01T00:00:00",
    "DocCreatedTimestampUtc": "2023-11-03T18:26:34Z",
    "QueueOffset": 0,
    "QueuePartition": 0
  },
  "SearchPatterns": {
    "SpaceKey": "Measlot::SPACEAGGED2:BAT:BE:BE1I:FOL_Autovision::ProcessControl:1.0",
    "TimeGroup": "0",
    "SiteKey": "BAT"
  }
}